src/evaluator.rs

```diff
@@ -136,7 +149,7 @@ mod tests {
         return_value: 101,
     }
 
-    evaluator_test_case! {
+    positive_evaluator_test_case! {
```

Comment:
maybe stick with `evaluator_test_case` and use `evaluator_error_test_case`?

Reply:
Done